### DISCUSSION TITLE
copilot not installed

### DISCUSSION THREAD

**Discussion by JESii** [community/community#30150] [Copilot Conversations] [answered]
I'm running NVIM v0.6.1 with node V14.17.0.
I'm using `vim-plug`
I added `Plug 'github/copilot.vim` to .vimrc
I run `PlugInstall`
There is no mention of copilot in the output -- it's never installed and I see no error message.
and `:Copilot setup` says there's no Copilot command.

What the heck is going on? There must be something I'm missing as I've been using nvim/vim-plug for quite some time now.

**Comment by JESii** [comment]
OK; so I upgraded to Neovim 0.7.2 and now copilot at least attempts to install... but errors with the following message:
```
x copilot.vim:
    fatal: invalid reference: master
```
All my other plugins work just fine...
What is the reference they are referring to here?

figured it out; need:
`Plug 'github/copilot.vim', {'branch': 'main'}`
and now it's installed

**Reply by tpope** [reply]
Old versions of vim-plug hard code `master`. Upgrade and you won't need to specify a branch.

**Comment by JESii** [comment]
Plug ‘copilot.vim, {‘branch’: ‘main’}
Solved the problem.

Jon Seidel, CMC®

**Comment by JESii** [comment]
Well... That was a temporary fix. Today I did a PlugUpdate and now I get the following error:
```
x copilot.vim:
    fatal: invalid reference: main
```
And if I remove `, {'branch':'main'}` then I'm back to the original failure complaining about `master`

**Reply by tpope** [reply]
Upgrade vim-plug.

**Comment by JESii** [comment, marked as answer]
Thanks @tpope; upgraded, removed the `('branch':'main'}` option, and then copilot updated successfully!

**Reply by 3kh0** [reply]
Please mark something as the answer!
